Library/Homebrew/manpages/brew.1.md

```diff
@@ -15,8 +15,7 @@ didn't include with OS X.
 
 For the full command list, see the COMMANDS section.
 
-With `--verbose` or `-v`, many commands print extra debugging information.
-Note that these flags should only appear after a command.
+With `--verbose` or `-v`, many commands print extra debugging information. With `--debug` or `-d`, additional debugging output and behaviors are enabled. Details are documented under the individual commands below. Note that these flags should only appear after a command.
```

Comment:
My bad. I hadn't taken into account the rest of the file when complaining about the formatting.